bandit/grammar.h

```diff
@@ -28,14 +28,13 @@ namespace bandit {
     listener.context_ended(desc);
   }
 
-  inline void describe(const std::string& desc, detail::voidfunc_t func) {
-    describe(desc, func, detail::registered_listener(), detail::context_stack());
+  inline void describe(const std::string& desc, detail::voidfunc_t func, bool hard_skip = false) {
```

Comment:
This is a breaking change because the signature of `describe` and `it` changed. Should be mentioned in the list of breaking changes.